test/cql-pytest/test_describe.py

```diff
@@ -2567,3 +2568,90 @@ def aux(row: DescRow) -> DescRow:
             assert restore_stmts == res
         finally:
             cql.execute(f"DROP KEYSPACE IF EXISTS {ks}")
+
+### ===========================================================================
+
+# `varchar` is an alias for `text`. Scylla defaults to `text` and we test it here.
+# It's not very important, but it's better to have a test that detects changes
+# like that and verifies we handle that case as expected.
+def test_restore_udf_with_varchar(cql, test_keyspace):
+    fn_content = """\
+        (arg {type_name})
+        RETURNS NULL ON NULL INPUT
+        RETURNS {type_name}
+        LANGUAGE lua
+        AS $$
+        return arg
+        $$;"""
+    # Get rid of indentation in the string.
+    fn_content = textwrap.dedent(fn_content)
```

Comment:
Ok, but why bother? What does the indentation hurt the test?

Reply:
If we dropped this line, the output of describe would be different because of the indentation and the test would fail

Reply:
I see. Ok. Maybe instead we could have "canonized" whitespace in both strings before comparing them (also these newlines can change, can't they?), but I guess that the dedent thing is good enough.